lib/block_view.go

```diff
@@ -3035,6 +3035,18 @@ func (bav *UtxoView) _connectUpdateGlobalParams(
 				return 0, 0, nil, fmt.Errorf("_connectUpdateGlobalParams: unable to decode LeaderScheduleMaxNumValidators as uint64")
 			}
 		}
+		if len(extraData[ValidatorSetMaxNumValidatorsKey]) > 0 {
```

Comment:
I think having these put in here before the "meta-epoch" level params makes sense.